core/injector/application_injector.hpp

```diff
@@ -593,6 +593,25 @@ namespace kagome::injector {
         injector.template create<sptr<application::ChainSpec>>();
 
     auto path = config.keystorePath(genesis_config->id());
+
+    if (config.isRunInDevMode()) {
+      boost::filesystem::create_directories(path);
+      std::ofstream ofs;
+
+      // Create predefined keys for 'Alice' user
+      // clang-format off
+      ofs.open((path / "babed43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d").native(), std::ios::ate);
```

Comment:
I think it would be useful to declare these keys as constants in a separate file, they might be useful in several places.